src/analysis/retail/shaman/restoration/modules/talents/CloudburstTotem.tsx

```diff
@@ -89,6 +95,49 @@ class CloudburstTotem extends Analyzer {
     this.eventEmitter.fabricateEvent(fabricatedEvent, event);
   }
 
+  /** Guide subsection describing the proper usage of Cloudburst Totem */
+  get guideSubsection(): JSX.Element {
+    const explanation = (
+      <p>
+        <b>
+          <SpellLink id={TALENTS.CLOUDBURST_TOTEM_TALENT.id} />
```

Comment:
Nit: change all usages of `<SpellLink id={..}>` to `<SpellLink spell={..}>` since we're trying to deprecate id